crates/ruff/src/rules/pycodestyle/rules/bare_except.rs

```diff
@@ -7,6 +7,32 @@ use crate::source_code::Locator;
 use crate::violation::Violation;
 
 define_violation!(
+    ///## What it does
+    ///Checks for bare `except:` statements in `try...except`.
+    ///
+    ///## Why is this bad?
+    ///A bare except catches `BaseExceptions` which includes `KeyboardInterrupt`, `SystemExit`, `Exception` and others. It can make it hard to interrupt the program with Ctrl+C and disguise other problems.
```

Comment:
please wrap the text

Reply:
> `BaseExceptions`

I think the plural s should be outside of the code tag.

Reply:
> please wrap the text (the `ruff rule` command directly prints it)

I'm not sure what you are asking for here.

Reply:
I meant that the line is very long and can be wrapped by inserting some line breaks. I don't think we want it to be longer than 80 chars.

Reply:
> > `BaseExceptions`
> 
> I think the plural s should be outside of the code tag.
 
 Fixed this.

Reply:
> the line is very long and can be wrapped by inserting some lin

Ahh, thanks. Are we okay with the links and code lines being longer than 80 characters?

Reply:
Yeah links being longer is fine. I think it would be nice if the code would fit in 80 chars but it's less important.

Reply:
Sounds good. I'll try to keep that in mind next time. I've got a ruler in VScode reminding me now =)

Reply:
Would love if we could automatically enforce the wrapping somehow for documentation specifically :sob: